packages/utils-node/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@linaria/core": "^5.0.2",
     "@linaria/react": "^5.0.3",
-    "@reapit/connect-session": "^6.1.1",
+    "@reapit/connect-session": "^6.1.3",
```

Comment:
probably need to remove connect-session from utils-node. It's about to cause a massive problem due to being removed from the monorepo. "self is not defined" will be the execption

Reply:
Yep. done